manifests/virt-manifest.yaml.in

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: virt-manifest-service
+spec:
+  ports:
+    - port: 8186
+      targetPort: virt-manifest
+  externalIPs:
+    - "{{ master_ip }}"
```

Comment:
On one hand it is nice that it is exposed here, for testing, on the other hand, it should not be accessible at the end. We probably sooner or later need to create different manifests, or maybe change that, for as soon as we make use of it through an explicit rest endpoint in our apiserver, or when using it for defaulting when posting VMs.